### ISSUE TITLE
[Announcement] New Model is Coming.

### ISSUE THREAD

**Issue by ildoonet** [ildoonet/tf-pose-estimation#150]
I'm going to improve this faster version of openpose using latest architectures.

Lots of issues have been filed up, I'm sorry that I didn't answer all the questions and handle issues.  

Basically, my research will be focused on improving both of accuracy and efficiency, for another 2 month. I will do my best to make a new model which is much better than current one.

Here are some keywords for the next architecture.

- Temporal Information : Many of you are using this code for video. I will add a new sub-architecture which will leverage temporal information to predict more efficiently and robustly.
- New Mobile-Friendly Architecture
- TensorRT, C++, Mobile Support

Thanks.

**Comment by ruimgf** [comment]
Are you think in launching a version with facial key points?

**Comment by ildoonet** [comment]
@ruimgf Not at this moment. I might add a plan for that but it is the top priority to improve accuracy(stability in video) and efficiency on the pose estimation model.

**Comment by khaerulumam42** [comment]
Nice @ildoonet , thank you!

**Comment by ouceduxzk** [comment]
@filipetrocadoferreira  that's not true. TensorRT can work on many nvidia gpu architectures, Jetson is one of them, you can check out nvidia blogs about how it works, it has python and c++ api currently. When you do inference, you can generate an engine for jetson or for volta

**Comment by filipetrocadoferreira** [comment]
You can, but the main improvements currently are targeted to Volta architectures. But the improvements for jetson can be significant (2~4x)

**Comment by atyshka** [comment]
@ildoonet What ever happened to this project?

**Comment by ildoonet** [comment]
@atyshka I have been quite busy. I re-started few weeks ago, so I should work hard to catch up :)

**Comment by atyshka** [comment]
Thanks for the new mobilenet architecture! Any updates on TensorRT?
